magefile.go

```diff
@@ -45,6 +49,13 @@ var (
 	}
 )
 
+// Aliases are shortcuts to long target names.
+// nolint: deadcode // it's used by `mage`.
+var Aliases = map[string]interface{}{
+	"llc":  mage.Linter.LastChange,
```

Comment:
Is there somewhere we should document the lint targets?

The developer guide doesn't really mention linting: https://www.elastic.co/guide/en/beats/devguide/current/index.html

I'm not even sure how up to date that documentation is. At least the CI will lead people towards the linter when the lint step fails.

Reply:
We try to update it from time to time. We should mention the new linter targets in the guide.